src/main/java/uk/gov/companieshouse/insolvency/delta/constants/CaseNumberDatesMap.java

```diff
@@ -28,7 +28,20 @@ public class CaseNumberDatesMap {
         }
     };
 
+    private static final Map<Integer, String> mandatoryCaseTypesField = new HashMap<>() {
```

Comment:
Maybe this could be called mandatoryFieldsForCaseType, make it a little more readable